myft-digest-promo/main.scss

```diff
@@ -16,38 +16,34 @@ $icon-width-m: 90px;
 	display: block;
 }
 
-.n-myft-digest-promo__wrapper {
-	position: relative;
-}
 .n-myft-digest-promo__container {
+	position: relative;
 	@include oGridRespondTo('M') {
 		@include oGridCenter;
-	}
-	margin-top: -25px;
-}
-
-.n-myft-digest-promo-dismiss-btn__container {
-		@include oGridRespondTo('M') {
-		@include oGridCenter;
+		@include oGridOffset(2);
 	}
 }
 
 .n-myft-digest-promo__dismiss-btn {
+	position: absolute;
+	top: 5px;
+	right: 0;
+	background-color: transparent;
+	border-color: transparent;
+
 	&::before {
 		@include oIconsGetIcon('cross-disk', getColor('grey-tint5'), 20);
+		position: absolute;
+		top: 0;
+		right: 0;
 		content: '';
-		top: 1px;
-		padding-top: 15px;
 	}
-	@include oGridOffset(4);
-	float: right;
-	background-color: transparent;
-	border-color: transparent;
 }
 
 .n-myft-digest-promo__content {
 	position: relative;
 	padding-left: $icon-width-s;
+	padding-top: 15px;
```

Comment:
This is a good opportunity to use our [slightly under-publicised spacing unit](https://github.com/Financial-Times/n-ui/blob/master/env.scss#L10)